redis/redis.go

```diff
@@ -16,6 +16,7 @@ type Options struct {
 	Port     int
 	User     string
 	Password secret.String
+	DB       int // must be 0-15
```

Comment:
I think you can [configure redis](https://raw.githubusercontent.com/redis/redis/5.0/redis.conf) with many more than 16 databases available, for the record

Reply:
ah, thanks for that I'll remove this comment